Sources/StreamChatUI/ChatThread/ChatThreadVC.swift

```diff
@@ -321,8 +331,16 @@ open class ChatThreadVC: _ViewController,
 
     private func updateMessages(with changes: [ListChange<ChatMessage>]) {
         messageListVC.setPreviousMessagesSnapshot(self.messages)
+
         let messages = getRepliesWithThreadRootMessage(from: messageController)
-        messageListVC.setNewMessagesSnapshot(messages)
+        let newMessages: [ChatMessage]
+        if let repliesBypass = repliesBypass {
+            newMessages = repliesBypass(messages)
+        } else {
+            newMessages = messages
+        }
+
+        messageListVC.setNewMessagesSnapshot(newMessages)
         messageListVC.updateMessages(with: changes)
```

Comment:
This is a problem. We are not filtering the `changes` and use it for some logic inside the `messageListVC` 🤔

Reply:
This is now fixed 🙂